modules/apps/29-fee/keeper/keeper.go

```diff
@@ -281,31 +281,31 @@ func (k Keeper) DeleteFeesInEscrow(ctx sdk.Context, packetID channeltypes.Packet
 	store.Delete(key)
 }
 
-// IterateChannelFeesInEscrow iterates over all the fees on the given channel currently escrowed and calls the provided callback
+// IteratePacketFeesInEscrow iterates over all the fees on the given channel currently escrowed and calls the provided callback
 // if the callback returns true, then iteration is stopped.
-func (k Keeper) IterateChannelFeesInEscrow(ctx sdk.Context, portID, channelID string, cb func(identifiedFee types.IdentifiedPacketFee) (stop bool)) {
+func (k Keeper) IteratePacketFeesInEscrow(ctx sdk.Context, portID, channelID string, cb func(packetFees types.PacketFees) (stop bool)) {
```

Comment:
Why was this function renamed?

Reply:
Aha, yes. Sorry for the confusion here. This stems from me adding [IterateIdentifiedChannelFeesInEscrow](IterateIdentifiedChannelFeesInEscrow) in #915. This was added to not break existing code and reduce the PR diff, but leads to confusion as there shouldn't be two iteration functions.

In this PR I've changed `IterateChannelFeesInEscrow` to `IteratePacketFeesInEscrow` - this difference being that this function now iterates `PacketFees` (multiple fees without packet ID) instead of `IdentifiedPacketFee` (singular with packet ID). I updated the naming as I thought it suited a bit better. 
The other iteration function can be removed after #957 and #967. 

Sorry about the weird diff here.

Reply:
Do we need identified packet iteration for `Add gRPC/cli for incentivized packets on a certain channel` in #871?

For `IteratePacketFeesInEscrow` this is for a specific channel? So maybe `IteratePacketFeesInEscrowForChannel` or `IteratePacketFeesInEscrowOnChannel`?

Reply:
Yea it'd need pagination. Good point

The function name sounds fine to me. I guess if the function iterated all packet fees it'd be titled `IterateAllPacketFees`

random sidenote, but thoughts on making our iteration functions private? I feel like only functions with `Get` should be public and if it starts with `Iterate` it should be private

Reply:
Yeah I agree, the less functions exposed the better I guess! I'd be happy to make the iterate functions private. We should do that in a follow up if so tho!